projboard/tests/test_client.py

```diff
@@ -143,3 +143,16 @@ def test_aboutpage(client):
     template_names = set(tmpl.origin.template_name for tmpl in response.templates)
     # And check if that 'about/about.html' is in the set.
     assert 'about/about.html' in template_names
+
+
+def test_login(client):
+    response = client.get('/accounts/login/')
+    assert response.status_code == 200
+    template_names = set(tmpl.origin.template_name for tmpl in response.templates)
+    assert 'login/login.html' in template_names
+
+
+def test_logout(client):
+    response = client.post('/accounts/logout/')
+    assert response.status_code == 302
+    assert response.url == '/'
```

Comment:
User should be logged it before it is logged out
Please see previous comment

Reply:
done